sketches/BLE_Scratch/robot.cpp

```diff
@@ -0,0 +1,50 @@
+#include "robot.h"
+#include <Servo.h>
+
+Robot::Robot(int rightWheelPin, int leftWheelPin) {
+  rightWheel = Servo();
+  leftWheel = Servo();
+  rightWheel.attach(rightWheelPin);
+  leftWheel.attach(leftWheelPin);
+  speed = 10;
+  // used to calibrate the middle point 
+  rightWheel.write(90);
+  leftWheel.write(90);
+}
```

Comment:
I usually see the constructor in the `.h`, I don't know which are the implication. I also prefer to use `this` to explicitly show which are the class member.
```suggestion
Robot::Robot(int rightWheelPin, int leftWheelPin) {
  this->rightWheel = Servo();
  this->leftWheel = Servo();
  this->rightWheel.attach(rightWheelPin);
  this->leftWheel.attach(leftWheelPin);
  this->speed = 10;
  // used to calibrate the middle point 
  this->rightWheel.write(90);
  this->leftWheel.write(90);
}
```